### DISCUSSION TITLE
update parameters in configmap

### DISCUSSION THREAD

**Discussion by nujev** [skupperproject/skupper#1162] [Q&A] [answered]
I have reviewed the configurations of the configmap called skupper-site, and I see properties that I want to modify, for example, disable or enable the console and the flow-collector

I see that I can put values ​​in true/false, but the question is how to apply these values, any ideas?
I have tried skupper update or skupper init but nothing changes

I can't do a skupper delete or some reinstallation because I already have services mounted on it, it's best not to affect what's already running

**Comment by grs** [comment, marked as answer]
This is a known limitation[1] and something we are planning to address as soon as we can.

[1] https://github.com/skupperproject/skupper/issues/58
